### PR TITLE
completed task 2

### PR DESCRIPTION
**Pull request by Filom3na**

### CODE REVIEW

File: __tests__/topics.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  
+  
+    
+      test('responds with topic objects containing required properties', () => {
+        return request(app)
+          .get('/api/topics')
+          .expect(200)
+          .then(({body}) => {  
+            const topics = body.topics
+            expect(topics.length).toBe(3)
```
Comment (2202Hannah) [inline __tests__/topics.test.js:23]: Good job checking the length of the array to make sure it isn't empty as you are asserting inside a forEach()

File: app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+const {getTopics} = require('./db/topics.controller');
+
+
+// const apiRouter = require("./topics.router");
```
Comment (2202Hannah) [inline app.js:6]: Please remove commented out code before merging to main

File: db/topics.controller.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const { fetchTopics } = require('../db/topics.model');
+
+  exports.getTopics = (req, res, next) => {
+    fetchTopics()
+      .then(topics => {
+        console.log(topics)
```
Comment (2202Hannah) [inline db/topics.controller.js:6]: Please remove console.log before merging to main

File: db/topics.model.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection')
+
+exports.fetchTopics = () => {
+    return db.query(` SELECT * FROM topics`)
+    .then(({rows}) => {
+        // console.log(rows)
```
Comment (2202Hannah) [inline db/topics.model.js:6]: Remove this console.log

File: db/topics.router.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+const apiRouter = require('express').Router();
```
Comment (2202Hannah) [inline db/topics.router.js:1]: Please remove this file as it is not currently used by your app

**Review by 2202Hannah — CHANGES_REQUESTED** [review]
Hi Filomena, please see my in-line comments and remove comments and console.logs before you merge to main. You can merge this to main when you are ready :) 

You could also include a 500 error handler in your app.js - you would not need to test for this.

File: db/topics.router.js (hunk 5 above)
Comment (itskatherine) [reply db/topics.router.js:1]: Still need to do this Filomena, can you get rid of `.Router()` on the end of the this line 
Once you've done this you can merge this branch

**Review by itskatherine — COMMENTED** [review]
Hi Filomena, small fix to look at - once you've done it you can merge this branch

**Pull request merged**